### PR TITLE
Conference hotfix

### PR DESCRIPTION
**Pull request by Fjandin**
- Fix missing stripe code in conference transformer
- rename files

### CODE REVIEW

File: src/app/api/transform/conference.0/route-conference-archived.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { eq } from "drizzle-orm"
 import { ConferenceEventArchivedPayload } from "@/contracts/events/conference"
 import { db } from "@/database"
 import { conferences } from "@/database/schemas"
+import { archiveProduct } from "@/lib/stripe/product"
```
Comment (suuunly) [inline src/app/api/transform/conference.0/route-conference-archived.ts:6]: Just a quick question, if this ought to reside in the lib folder on the server side?
Or are they inaccessible regardless after the build, through NextJS magic? :)

Comment (Fjandin) [reply src/app/api/transform/conference.0/route-conference-archived.ts:6]: They should be inaccessible. I don't think the new nextjs uses folder structure to decide what it exposed and what is not.

**Review by suuunly — APPROVED** [review]
lgtm! A little confused by the Admin text in the Conference page. but given that it is all WIP, and a final design is not set yet. I think it is perfectly fine :D

**Comment by suuunly** [comment]
> > I noticed that when I tried to `yarn install`, I had to upgrade my node version. Maybe this should be mentioned in the readme. I am currently on 20.1.0. I switched to 20.11.0, and then it worked :)
> > The package in question, is `sharp`
> 
> hm... I haven't added any `sharp`

no worries. I will create an issue for that :)

**Comment by suuunly** [comment]
issue #77 created 😄

**Pull request merged**